solidity/test/vault/TBTCVault.test.ts

```diff
@@ -128,7 +128,7 @@ describe("TBTCVault", () => {
             account1.address,
             to1e18(800)
           )
-        ).to.be.revertedWith("Caller is not the governance")
+        ).to.be.revertedWith("Ownable: caller is not the owner")
```

Comment:
Would be good to change all contexts' names from `when called not by the governance` to `when called not by the owner`

Reply:
I was thinking about it but the owner of the vault is still governance (DAO/council). It's just not the intermediary governance contract.